yesod-form/Yesod/Form/I18n/Romanian.hs

```diff
@@ -0,0 +1,31 @@
+{-# LANGUAGE OverloadedStrings #-}
+module Yesod.Form.I18n.Romanian where
+
+import Yesod.Form.Types (FormMessage (..))
+import Data.Monoid (mappend)
+import Data.Text (Text)
+
+-- |
+-- Romanian translation
```

Comment:
Why are these separate lines? I would expect to see:

```suggestion
-- | Romanian translation
--
```

Reply:
Fixed (fdfac8de6f270f2eefaf44b90cae766bbca3c936).